{
	"id": "openSUSE-SU-2020:1002-1",
	"modified": "2020-07-18T22:27:00Z",
	"published": "2020-07-18T22:27:00Z",
	"related": [
		"CVE-2020-14422"
	],
	"summary": "Security update for python-ipaddress",
	"details": "This update for python-ipaddress fixes the following issues:\n\n- Add CVE-2020-14422-ipaddress-hash-collision.patch fixing\n  CVE-2020-14422 (bsc#1173274, bpo#41004), where hash collisions\n  in IPv4Interface and IPv6Interface could lead to DOS. \n\nThis update was imported from the SUSE:SLE-15:Update update project.",
	"affected": [
		{
			"package": {
				"ecosystem": "openSUSE:Leap 15.2",
				"name": "python-ipaddress",
				"purl": "purl:rpm/suse/python-ipaddress&distro=openSUSE%20Leap%2015.2"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "1.0.18-lp152.4.3.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python-ipaddress": "1.0.18-lp152.4.3.1"
					}
				]
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/thread/W3G3MHKIU5FV6NKK5GS7XEYQ6W3XV56Z/"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.suse.com/1173274"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2020-14422"
		}
	]
}
